warehouse/query-core/src/main/java/datawave/query/jexl/visitors/BaseIndexExpansionVisitor.java

```diff
@@ -284,7 +285,11 @@ protected void logResult(String stage, JexlNode node, IndexLookupMap lookupMap)
             } else if (lookupMap.get(field).isThresholdExceeded()) {
                 log.debug("{} expansion for term [{}] failed (threshold)", stage, term);
             } else {
-                log.debug("{} expansion for term [{}] success ({} values)", stage, term, lookupMap.get(field).size());
+                try {
+                    log.debug("{} expansion for term [{}] success ({} values)", stage, term, lookupMap.get(field).size());
```

Comment:
The exception we hit was originating from the `lookupMap.containsKey()` call above. We might want to wrap the from like 281 down in the try/catch.